lib/icinga/apievents.cpp

```diff
@@ -234,14 +250,15 @@ void ApiEvents::AcknowledgementClearedHandler(const Checkable::Ptr& checkable, c
 		queue->ProcessEvent(result);
 	}
 
-	result->Set("acknowledgement_type", AcknowledgementNone);
+	inboxes.Push(std::move(result));
```

Comment:
That diff looks odd. Intended?

Reply:
There's no `acknowledgement_type` for `AcknowledgementCleared` events (see docs).

Reply:
Then the docs are wrong, I wrote them years after the implementation :)